### PR TITLE
Better node labels for calc-language expression trees

### PR DESCRIPTION
**Pull request by fgamador**

### CODE REVIEW

File: d3/query-graphs.js [hunk 1]
```diff
@@ -451,6 +451,36 @@ function prepareTreeData(_treeData, convertHyPer) {
...
+        }
+
+        // extensions for calculation-language expression trees
+        function handleExpression(node) {
+            if (node.text) {
+                node.name = node.text;
+            } else if (node.properties && node.properties.name) {
+                node.name = node.properties.name;
+            } else if (node.properties && node.properties.value) {
+                if (node.properties.type == "string") {
```
Comment (rickcole) [inline d3/query-graphs.js]: npm run lint will error on use of '=='

File: d3/query-graphs.js [hunk 2]
```diff
@@ -451,6 +451,36 @@ function prepareTreeData(_treeData, convertHyPer) {
             }
         }
 
+        // for calculation-language expression trees
+        function handleDimensions(node) {
```
Comment (rickcole) [inline d3/query-graphs.js]: Would be nice to add example query that benefits to media/favorites.

File: d3/query-graphs.js [hunk 3]
```diff
@@ -451,6 +451,36 @@ function prepareTreeData(_treeData, convertHyPer) {
...
+                node.name = node.text;
+            } else if (node.properties && node.properties.type) {
+                node.name = node.properties.type;
+            } else {
+                node.name = node.tag;
+            }
+        }
+
+        // extensions for calculation-language expression trees
+        function handleExpression(node) {
```
Comment (rickcole) [inline d3/query-graphs.js]: Would be nice to add query that benefits to media/favorites.

**Comment by fgamador** [comment]
Fixed the lint problem and added an example calc-expression-node tree. Not technically a query, but still works well.

**Pull request merged**